server/protocols/jmap-rfc-8621-integration-tests/jmap-rfc-8621-integration-tests-common/src/main/scala/org/apache/james/jmap/rfc8621/contract/MailboxSetMethodContract.scala

```diff
@@ -58,6 +58,7 @@ trait MailboxSetMethodContract {
   }
 
   def randomMailboxId: MailboxId
+  def errorInvalidMailboxIdMessage: String
```

Comment:
Not sure if this is ok or if overriding the test method in `DistributedMailboxSetMethodTest` would be better?